src/Coin.cpp

```diff
@@ -40,6 +41,7 @@
 #include "Wanchain/Address.h"
 #include "Waves/Address.h"
 #include "Zcash/TAddress.h"
+#include "IoTeX/Address.h"
```

Comment:
duplicated import again, why don't you look at the diff by yourself?

Reply:
it maybe git merge issue.